src/Microsoft.Extensions.Configuration.AzureAppConfiguration/TracingUtils.cs

```diff
@@ -18,32 +15,11 @@ internal static class TracingUtils
     {
         static private HostType? _hostType = null;
 
-        public static string GenerateUserAgent(string currentUserAgent = null)
+        public static string GenerateUserAgent()
```

Comment:
Do we call this anywhere else or can we just move it into the UserAgentHeaderPolicy?

Reply:
I guess we use it in tests, but could always make it internal so it's visible to tests.

Reply:
We create an instance of the class `UserAgentHeaderPolicy` and send it to the SDK to include it in its HTTP pipeline. It currently only has methods that override the base class methods. From the design perspective, I'm not sure if we should include utility methods in the same class. Perhaps, that was the original reasoning this code was extracted out in the first place, but let me know if you think otherwise.

Reply:
Having `GenerateUserAgent` in a utility method doesn't sound right to me. I don't see a problem moving it into `UserAgentHeaderPolicy` even though right now it just overrides base class methods.